sql/item_mods.sql

```diff
@@ -19998,16 +19998,40 @@ INSERT INTO `item_mods` VALUES (16397,25,3);
 INSERT INTO `item_mods` VALUES (16398,25,3);
 INSERT INTO `item_mods` VALUES (16398,431,1);
 INSERT INTO `item_mods` VALUES (16403,431,1);
+INSERT INTO `item_mods` VALUES (16404,431,1);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,499,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,501,10);  -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,951,3);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,952,4);   -- Venom Katars - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16404,953,30);  -- Venom Katars - Additional effect: Poison
 INSERT INTO `item_mods` VALUES (16410,431,1);
 INSERT INTO `item_mods` VALUES (16416,54,5);
+INSERT INTO `item_mods` VALUES (16416,431,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,499,1);   -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,500,30);  -- Dragon Claws - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16416,501,10);  -- Dragon Claws - Additional effect: Fire damage
 INSERT INTO `item_mods` VALUES (16417,431,1);
+INSERT INTO `item_mods` VALUES (16418,431,1);   -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,499,10);  -- Venom Claws - Additional effect: Poison
+INSERT INTO `item_mods` VALUES (16418,501,10);  -- Venom Claws - Additional effect: Poison
```

Comment:
I'm not fond of us just repeating the same comment, it doesn't aid anyone in knowing what the line does. Thats why i was writing my examples the way I did: it told you exactly what that line did, by itself